pydoll/browser/options.py

```diff
@@ -95,3 +122,192 @@ def add_argument(self, argument: str):
             self._arguments.append(argument)
         else:
             raise ArgumentAlreadyExistsInOptions(f'Argument already exists: {argument}')
+
+    @property
+    def prefs_options(self) -> dict:
+        return self._prefs_options
+
+    @prefs_options.setter
+    def prefs_options(self, prefs: dict):
+        if not isinstance(prefs, dict):
+            raise ValueError("The experimental options value must be a dict.")
+        if prefs.get('prefs'):
+            raise WrongPrefsDict
+        self._prefs_options = {**self._prefs_options, **prefs}
+
+    def set_pref_path(self, path: list, value):
```

Comment:
I think this method should be private: 
```suggestion
    def _set_pref_path(self, path: list, value):
```